### PR TITLE
Brands page and brands_apple test are added

### PR DESCRIPTION
**Pull request by mistergreen22**
The page with all brands are added in this commit, and test which check redirecting to the apple brand page from the all brands page are implemented.

### CODE REVIEW

File: oct/pages/brands.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    def __init__(self, browser: Remote):
+        self._browser = browser
+
+    def open(self) -> None:
+        self._browser.get("https://localhost/index.php?route=product/manufacturer")
+
+    def loaded(self) -> bool:
+        return "Find Your Favorite Brand" in self._browser.title
+
+    def click_apple_brand(self) -> None:
```
Comment (AlekseyDidilov) [inline oct/pages/brands.py]: @mistergreen22 Why you hardcoded brand, I suggest you add brand in constructor.

Comment (mistergreen22) [reply oct/pages/brands.py]: @AlekseyDidilov  thanks, i made changes.

File: oct/tests/web/all_brands.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+from oct.pages.brands import BrandsPage
+
+
+class AllBrands(Testcase):
+    @test
+    def test(self, grid: str) -> None:
+        chrome: Remote = Chrome(grid)
+        brands_page = BrandsPage(chrome, 1)
+        brands_page.open()
+        brands_page.loaded()
```
Comment (extsoft) [inline oct/tests/web/all_brands.py]: @mistergreen22 why do you call this method?

File: oct/tests/web/all_brands.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        brands_page.click_brand_name()
```
Comment (extsoft) [inline oct/tests/web/all_brands.py]: @mistergreen22 please pass here which brand to choose.

File: oct/tests/web/all_brands.py (hunk 2 above)
Comment (mistergreen22) [reply oct/tests/web/all_brands.py]: @extsoft removed, it's redundant.

File: oct/tests/web/all_brands.py (hunk 3 above)
Comment (mistergreen22) [reply oct/tests/web/all_brands.py]: @extsoft done.

Comment (extsoft) [reply oct/tests/web/all_brands.py]: @mistergreen22 you need to pass a brand in this line.

Comment (mistergreen22) [reply oct/tests/web/all_brands.py]: @extsoft get it, now i pass brand like an argument in this method
`brands_page.click_brand_name(BrandList.APPLE)`

**Pull request merged**